nemo/collections/common/prompts/gemma.py

```diff
@@ -37,7 +37,7 @@ class GemmaPromptFormatter(PromptFormatter):
 
 
 @registered_prompt_format_fn
-def gemma1(cuts: CutSet, tokenizer: TokenizerSpec):
+def gemma(cuts: CutSet, tokenizer: TokenizerSpec):
     prompt = GemmaPromptFormatter(tokenizer)
     ans = defaultdict(list)
     for cut in cuts:
```

Comment:
below (l.54) there's a system prompt support that should be removed - this was copied from llama2, but gemma doesn't have a system turn.